### PR TITLE
Cleanup skopeo man page and README.md

### PR DESCRIPTION
**Pull request by rhatdan**
Fix spelling mistakes
Fix reference to garbage collection on the container registry server.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@runcom @jwhonce PTAL

**Comment by rhatdan** [comment]
@runcom I think we need to improve the README to deemphasize atomic registries, but mention other registries like AWS and google registries, perhaps other vendors besides docker.  Aritfactory, OpenShift, Quay.io, others?

**Comment by rhatdan** [comment]
@ipbabble PTAL

**Comment by rhatdan** [comment]
@TomSweeneyRedHat PTAL

**Comment by rhatdan** [comment]
We probably need to mention more prominently the use of github.com/containers/image and to a lesser degreee containers/storage.

File: docs/skopeo.1.md [hunk 1]
```diff
@@ -2,13 +2,25 @@
...
+`skopeo` is a command line utility providing various operations with container images and container image registries.
+
+`skopeo` can copy container images between various containers image stores, possibly converting them as necessary.  For example you can use `skopeo` to copy container images from one container registry to another.
+
+`skopeo` can convert a Docker v2.1 container image to an OCI image.
+
+`skopeo` can inspect a repository on a container registry without pulling the image. It fetches the repository's manifest and displays a `docker inspect`-like json output about a whole repository or a tag. `skopeo`, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? Which labels the image has?
+
+`skopeo` can also sign and verify container images.
```
Comment (TomSweeneyRedHat) [inline docs/skopeo.1.md]: I think I'd drop 'also' in this line.

File: docs/skopeo.1.md [hunk 2]
```diff
@@ -2,13 +2,25 @@
...
+skopeo -- Command line utility used to interact with local and remote container images and container image registries
 # SYNOPSIS
 **skopeo** [_global options_] _command_ [_command options_]
 # DESCRIPTION
-`skopeo` is a command line utility providing various operations with container images and container image registries. For example, it is able to inspect a repository on a container registry and fetch image. It fetches the repository's manifest and it is able to show you a `docker inspect`-like json output about a whole repository or a tag. This tool, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? which labels the image has?
+`skopeo` is a command line utility providing various operations with container images and container image registries.
+
+`skopeo` can copy container images between various containers image stores, possibly converting them as necessary.  For example you can use `skopeo` to copy container images from one container registry to another.
```
Comment (TomSweeneyRedHat) [inline docs/skopeo.1.md]: I'd drop 'possibly'.

File: docs/skopeo.1.md [hunk 3]
```diff
@@ -2,13 +2,25 @@
...
+
+`skopeo` can convert a Docker v2.1 container image to an OCI image.
+
+`skopeo` can inspect a repository on a container registry without pulling the image. It fetches the repository's manifest and displays a `docker inspect`-like json output about a whole repository or a tag. `skopeo`, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? Which labels the image has?
```
Comment (TomSweeneyRedHat) [inline docs/skopeo.1.md]: I'd suggest "about a whole repository" to "about the repository".  

I think I'd cap "which tags" to "Which tags".

Change "Which labels the image has?" to "Which labels does the image have?"

File: docs/skopeo.1.md [hunk 4]
```diff
@@ -91,10 +103,17 @@ Existing signatures, if any, are preserved as well.
 ## skopeo delete
 **skopeo delete** _image-name_
 
-Mark _image-name_ for deletion.  To release the allocated disk space, you need to execute the container registry garabage collector. E.g.,
+Mark _image-name_ for deletion.  To release the allocated disk space, you need login to the container registry server and execute the container registry garbage collector. E.g.,
```
Comment (TomSweeneyRedHat) [inline docs/skopeo.1.md]: 'you need login to" to "you need to login into"

File: docs/skopeo.1.md [hunk 5]
```diff
@@ -91,10 +103,17 @@ Existing signatures, if any, are preserved as well.
...
+
+```
+/usr/bin/registry garbage-collect /etc/docker-distribution/registry/config.yml
+
+Note: sometimes the config.yml is stored in /etc/docker/registry/config.yml
+
+If you are running the container registry inside of a container you would execute something like
```
Comment (TomSweeneyRedHat) [inline docs/skopeo.1.md]: ':" at the end of like?

File: docs/skopeo.1.md [hunk 6]
```diff
@@ -22,7 +34,7 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$HOME/.docker/config.json`, which is set e.g. using `(docker login)`.
```
Comment (TomSweeneyRedHat) [inline docs/skopeo.1.md]: This part about the default auth location is changing with @umohnani8's submit or do you want to touch it up now?  If not changing, at least for the moment, I'd remove the 'e.g.' here.

File: README.md [hunk 7]
```diff
@@ -5,10 +5,15 @@ skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=ma
 
 ----
 
-`skopeo` is a command line utility that performs various operations on container images and image repositories.  Skopeo works with API V2 registries such as Docker registries, the Atomic registry, private registries, local directories and local OCI-layout directories.  Skopeo does not require a daemon to be running to perform these operations which consist of:
+`skopeo` is a command line utility that performs various operations on container images and image repositories.
+
+`skopeo` can work with (OCI images)[https://github.com/opencontainers/image-spec] as well as the original Docker v2 images.
+
+Skopeo works with API V2 registries such as Docker registries, the Atomic registry, private registries, local directories and local OCI-layout directories.  Skopeo does not require a daemon to be running to perform these operations which consist of:
```
Comment (runcom) [inline README.md]: Yup let's mention GCR, Artifactory as well Dan

File: README.md [hunk 8]
```diff
@@ -5,10 +5,15 @@ skopeo [![Build Status](https://travis-ci.org/projectatomic/skopeo.svg?branch=ma
...
 
- * Inspecting an image showing its properties including its layers.
  * Copying an image from and to various storage mechanisms.
+   For example you can copy images from one registry to another.
```
Comment (runcom) [inline README.md]: I'd say also w/o needing any privilege

File: docs/skopeo.1.md (hunk 3 above)
Comment (runcom) [reply docs/skopeo.1.md]: Also "without pulling the image" -> "without pulling a specific tag from the repository"

File: docs/skopeo.1.md [hunk 9]
```diff
@@ -2,13 +2,25 @@
...
+`skopeo` can convert a Docker v2.1 container image to an OCI image.
+
+`skopeo` can inspect a repository on a container registry without pulling the image. It fetches the repository's manifest and displays a `docker inspect`-like json output about a whole repository or a tag. `skopeo`, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? Which labels the image has?
+
+`skopeo` can also sign and verify container images.
+
+`skopeo` can delete container images from a remote container registry.
+
+Note: `skopeo` does not require any container runtimes to be running, to do most of
+its functionality.
```
Comment (runcom) [inline docs/skopeo.1.md]: And doesn't need root unless you're dealing with the Docker daemon or storage transport

File: docs/skopeo.1.md (hunk 1 above)
Comment (ipbabble) [reply docs/skopeo.1.md]: Just a couple of thoughts. I like being explicit as to why pulling is undesirable. Assume n00bs.

s/registry without pulling/registry without needlessly pulling/ 
s/image. It fetches/image. Pulling an image from a repository, especially a remote repository, is an expensive network and storage operation. Skopeo fetches/

File: docs/skopeo.1.md (hunk 2 above)
Comment (ipbabble) [reply docs/skopeo.1.md]: +1

File: docs/skopeo.1.md (hunk 1 above)
Comment (rhatdan) [reply docs/skopeo.1.md]: Fixed

File: docs/skopeo.1.md (hunk 2 above)
Comment (rhatdan) [reply docs/skopeo.1.md]: Fixed

File: docs/skopeo.1.md (hunk 3 above)
Comment (rhatdan) [reply docs/skopeo.1.md]: I Fixed @TomSweeneyRedHat suggestions.
@runcom I think yours might be a little too technical.  I already changed that line with @ipbabble response `without needlessly pulling the image`.

BTW Once we have this cleaned up, I would like to write a blog using this content to reintroduce skopeo.

File: docs/skopeo.1.md (hunk 4 above)
Comment (rhatdan) [reply docs/skopeo.1.md]: Changed to `you must login to`

File: docs/skopeo.1.md (hunk 5 above)
Comment (rhatdan) [reply docs/skopeo.1.md]: Fixed

File: docs/skopeo.1.md (hunk 6 above)
Comment (rhatdan) [reply docs/skopeo.1.md]: I think we should add `kpod login" AFTER we have completed the rename and introduce it.

Comment (TomSweeneyRedHat) [reply docs/skopeo.1.md]: That's probably best.

File: docs/skopeo.1.md (hunk 4 above)
Comment (TomSweeneyRedHat) [reply docs/skopeo.1.md]: That's better yet.

File: README.md (hunk 7 above)
Comment (rhatdan) [reply README.md]: Fixed

File: README.md (hunk 8 above)
Comment (rhatdan) [reply README.md]: Added

File: docs/skopeo.1.md (hunk 9 above)
Comment (rhatdan) [reply docs/skopeo.1.md]: fixed

**Comment by rhatdan** [comment]
@runcom @mtrmac PTAL

**Comment by runcom** [comment]
LGTM

needs rebase @rhatdan, @mtrmac PTAL

**Comment by rhatdan** [comment]
@runcom rebased

File: docs/skopeo.1.md [hunk 10]
```diff
@@ -2,13 +2,25 @@
...
+skopeo -- Command line utility used to interact with local and remote container images and container image registries
 # SYNOPSIS
 **skopeo** [_global options_] _command_ [_command options_]
 # DESCRIPTION
-`skopeo` is a command line utility providing various operations with container images and container image registries. For example, it is able to inspect a repository on a container registry and fetch image. It fetches the repository's manifest and it is able to show you a `docker inspect`-like json output about a whole repository or a tag. This tool, in contrast to `docker inspect`, helps you gather useful information about a repository or a tag without requiring you to run `docker pull` - e.g. - which tags are available for the given repository? which labels the image has?
+`skopeo` is a command line utility providing various operations with container images and container image registries.
+
+`skopeo` can copy container images between various containers image stores, converting them as necessary.  For example you can use `skopeo` to copy container images from one container registry to another.
+
+`skopeo` can convert a Docker v2.1 container image to an OCI image.
```
Comment (mtrmac) [inline docs/skopeo.1.md]: This probably refers  (API version 2, not really a part of the image); schema 1, or schema 2. I haven’t seen a dot notation used for this. Maybe just “Docker container image“, or “Docker schema1 or schema2 container image“?

Comment (rhatdan) [reply docs/skopeo.1.md]: Fixed

**Comment by rhatdan** [comment]
@runcom @mtrmac fixed, I think this is ready to merge.

**Comment by mtrmac** [comment]
LGTM

**Comment by runcom** [comment]
I'll cut a new release now

**Pull request merged**